### DISCUSSION TITLE
Abandoned?

### DISCUSSION THREAD

**Discussion by amo13** [astubenbord/paperless-mobile#445] [General]
With complete silence from the developer since January, this project looks abandoned. The devs [last message](https://github.com/astubenbord/paperless-mobile/pull/376#issuecomment-2285906942) reads like he could need some help with keeping up.
Has anyone already started to maintain a fork? Maybe inside the paperless-ngx github community organization would be a good place to create one. Unfortunately, I have zero experience in mobile development so I can't really help with that. But it would be awesome to see this app come to life again.

**Comment by astubenbord** [comment]
I wouldnt say iy's abandoned, I'm locally still working on a larger update occasionally. But its just not possible to keep up with the amount of tickets, bugs and generally the speed at which paperless-ngx evolves on my own. Also I don't have as much time anymore as I had a year ago (university vs. full time job).

**Reply by amo13** [reply]
Thank you for your reply. And glad to read that you're still on it occasionally.
Maybe finding a co-maintainer or two would help bring your amazing app forward. I'm sure you can find good candidates if you just ask for it, like at the top of this repo's readme and with a discussion thread on the main ngx repo.
Opening the project to the community is exactly what saved ngx itself after it [died](https://github.com/the-paperless-project/paperless) [twice](https://github.com/jonaswinkler/paperless-ng).

**Comment by bingobob** [comment]
Just wanted to say, that I love this app and the whole idea with self hosted paperless. Great work. If a new update comes out, I'll be very happy.

The existing app is super great but feels like it's lagging and a bit unloved.

None the less. Great work and thanks.
